<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      
      <!-- Page 0: Login -->
      <widget class="QWidget" name="loginPage">
       <layout class="QVBoxLayout" name="loginLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="loginTitle">
          <property name="text">
           <string>üîê Admin Login</string>
          </property>
          <property name="styleSheet">
           <string>font-size: 28px; font-weight: bold; color: #667eea;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer1">
          <property name="orientation"><enum>Qt::Vertical</enum></property>
         </spacer>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="usernameLabel"><property name="text"><string>Username:</string></property></widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="usernameEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="passwordLabel"><property name="text"><string>Password:</string></property></widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="passwordEdit">
            <property name="echoMode"><enum>QLineEdit::Password</enum></property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="loginButton">
          <property name="text"><string>Login</string></property>
          <property name="minimumHeight"><number>50</number></property>
         </widget>
        </item>
        <item>
         <spacer name="spacer2">
          <property name="orientation"><enum>Qt::Vertical</enum></property>
         </spacer>
        </item>
       </layout>
      </widget>
      
      <!-- Page 1: Menu -->
      <widget class="QWidget" name="menuPage">
       <layout class="QVBoxLayout" name="menuLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="menuTitle">
          <property name="text"><string>Admin Menu</string></property>
          <property name="styleSheet">
           <string>font-size: 24px; font-weight: bold; color: #667eea;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addQuestionButton">
          <property name="text"><string>‚ûï Add Question</string></property>
          <property name="minimumHeight"><number>60</number></property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="viewQuestionsButton">
          <property name="text"><string>üìã View Questions</string></property>
          <property name="minimumHeight"><number>60</number></property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updateQuestionButton">
          <property name="text"><string>‚úèÔ∏è Update Question</string></property>
          <property name="minimumHeight"><number>60</number></property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteQuestionButton">
          <property name="text"><string>üóëÔ∏è Delete Question</string></property>
          <property name="minimumHeight"><number>60</number></property>
         </widget>
        </item>
        <item>
         <spacer name="spacer3">
          <property name="orientation"><enum>Qt::Vertical</enum></property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="logoutButton">
          <property name="text"><string>üö™ Logout</string></property>
          <property name="minimumHeight"><number>50</number></property>
         </widget>
        </item>
       </layout>
      </widget>
      
      <!-- Page 2: Add Question -->
      <widget class="QWidget" name="addPage">
       <layout class="QVBoxLayout" name="addLayout">
        <item>
         <widget class="QLabel" name="addTitle">
          <property name="text"><string>Add Question</string></property>
          <property name="styleSheet">
           <string>font-size: 24px; font-weight: bold; color: #27ae60;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="addForm">
          <item row="0" column="0"><widget class="QLabel" name="idLabel"><property name="text"><string>ID:</string></property></widget></item>
          <item row="0" column="1"><widget class="QLineEdit" name="idEdit"/></item>
          <item row="1" column="0"><widget class="QLabel" name="textLabel"><property name="text"><string>Question:</string></property></widget></item>
          <item row="1" column="1"><widget class="QTextEdit" name="textEdit"><property name="maximumHeight"><number>100</number></property></widget></item>
          <item row="2" column="0"><widget class="QLabel" name="optALabel"><property name="text"><string>Option A:</string></property></widget></item>
          <item row="2" column="1"><widget class="QLineEdit" name="optAEdit"/></item>
          <item row="3" column="0"><widget class="QLabel" name="optBLabel"><property name="text"><string>Option B:</string></property></widget></item>
          <item row="3" column="1"><widget class="QLineEdit" name="optBEdit"/></item>
          <item row="4" column="0"><widget class="QLabel" name="optCLabel"><property name="text"><string>Option C:</string></property></widget></item>
          <item row="4" column="1"><widget class="QLineEdit" name="optCEdit"/></item>
          <item row="5" column="0"><widget class="QLabel" name="optDLabel"><property name="text"><string>Option D:</string></property></widget></item>
          <item row="5" column="1"><widget class="QLineEdit" name="optDEdit"/></item>
          <item row="6" column="0"><widget class="QLabel" name="answerLabel"><property name="text"><string>Answer:</string></property></widget></item>
          <item row="6" column="1"><widget class="QComboBox" name="answerCombo"/></item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttonLayout">
          <item>
           <widget class="QPushButton" name="saveQuestionButton">
            <property name="text"><string>üíæ Save</string></property>
            <property name="minimumHeight"><number>50</number></property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancelButton">
            <property name="text"><string>‚ùå Cancel</string></property>
            <property name="minimumHeight"><number>50</number></property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      
      <!-- Page 3: View Questions -->
      <widget class="QWidget" name="viewPage">
       <layout class="QVBoxLayout" name="viewLayout">
        <item>
         <widget class="QLabel" name="viewTitle">
          <property name="text"><string>View Questions</string></property>
          <property name="styleSheet">
           <string>font-size: 24px; font-weight: bold; color: #3498db;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="countLabel">
          <property name="text"><string>Total: 0 questions</string></property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="questionsList"/>
        </item>
        <item>
         <widget class="QPushButton" name="backToMenuButton">
          <property name="text"><string>‚Üê Back</string></property>
          <property name="minimumHeight"><number>50</number></property>
         </widget>
        </item>
       </layout>
      </widget>
      
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
